cmd/main.go

```diff
@@ -1,5 +1,39 @@
 package main
 
+import (
+	"context"
+	"net/http"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/ElrondNetwork/notifier-go/config"
+	"github.com/ElrondNetwork/notifier-go/proxy"
+)
+
 func main() {
+	cfg, err := config.LoadConfig()
+	if err != nil {
+		panic(err)
+	}
+
+	api, err := proxy.NewWebServer(cfg)
+	if err != nil {
+		panic(err)
+	}
+	server := api.Run()
+	waitForGracefulShutdown(server)
+}
+
+func waitForGracefulShutdown(server *http.Server) {
+	quit := make(chan os.Signal)
+	signal.Notify(quit, os.Interrupt, os.Kill)
+	<-quit
 
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
make '5' a constant variable and use that in the code

Reply:
done